src/content/faq/General.md

```diff
@@ -9,70 +9,71 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
-<!-- TODO -->
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance and stability by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI clients](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
+* Create a representation of your data that feels familiar and natural ([a graph](/learn/thinking-in-graphs/#it-s-graphs-all-the-way-down-https-en-wikipedia-org-wiki-turtles-all-the-way-down)).
+* Ensure that all of your data is [statically typed](/learn/schema/) and these types inform [what queries the schema supports](/learn/introspection/).
+* Examine queries and responses through community tools like [GraphiQL](https://github.com/graphql/graphiql).
+
+And many others! [Our homepage](/) outlines some more convincing reasons to use GraphQL.
+
+Additionally, it doesn’t take rewriting your entire application to try out GraphQL. You can even start with a single HTTP request that wraps an existing REST call, for instance. Your [GraphQL schema](/learn/thinking-in-graphs/#shared-language) and [business domain model](/learn/thinking-in-graphs/#business-logic-layer) can be expanded gradually, so we’d recommend focusing on one use case at first and only build the part of the schema you need for that to work. 
 
-Soufflé candy lollipop [dragée cake chocolate](). Marzipan lemon drops sweet roll I love cake ice cream muffin chupa chups gummi bears. Ice cream fruitcake I love sweet roll croissant ice cream. Gummi bears sesame snaps cake apple pie cake jelly. Gingerbread powder I love apple pie marzipan jelly beans sugar plum sesame snaps. Tiramisu chupa chups I love bear claw cake. Macaroon muffin topping ice cream bear claw cookie I love marshmallow. Pie gummies toffee candy pastry soufflé. Chupa chups toffee gummi bears I love cake lemon drops.
 
 ## Is GraphQL a database language like SQL?
```

Comment:
@IvanGoncharov Should we address the misconception that GraphQL only works with databases that are graph-based? Or save that for a separate question?

Reply:
I think we can address it by changing:

> GraphQL is designed for APIs, not databases.

into something like: GraphQL is designed to work any data source exposed through API, not only database.